modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -0,0 +1,108 @@
+= Create an API Using the CLI
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, downloading, installing
+
+After you provision the adapter, you must create an API to bind to the service. The following diagram explains the flow for creating an API:
+
+//image::configure-service-mesh-flowchart-2.png[]
+image::create-API-CLI-flowchart.png[]
+
+You can create an API using Anypoint Platform or Anypoint Service Mesh:
+
+* <<create-api-AP, Create an API using Anypoint Platform>>
++
+If you want to create REST or SOAP APIs with OAS or WSDL definition, use Anypoint Platform. You can manage your contracts and complex API configurations easily with Anypoint Platform.
++
+Note that if you want to upload a REST API or OAS asset type, you upload a .json or .yaml file. Alternatively, if you want to upload a SOAP API or WSDL asset type, you upload a .wsdl or .xml file.
+* <<create-api-ASM-CLI, Create an API using Anypoint Service Mesh Autocreate>>
```

Comment:
@priyaayyar This header looks really odd. Please change it to 

> Create an API using Anypoint Service Mesh (Auto-create)

Reply:
Went with Create an API using Anypoint Service Mesh